valohai_yaml/objs/pipelines/node.py

```diff
@@ -20,14 +20,19 @@ class Node(Item):
     # `actions` will be set on instance level in subclasses
     actions: List[NodeAction]
 
+    # `on_error` what the pipeline should do when this node is erroneous: "stop-all" default, "continue"
+    on_error: str
```

Comment:
I think the in-memory representation for this should be an Enum, like `KeepDirectories` and `MultipleMode` are.